src/plone/restapi/serializer/relationfield.py

```diff
@@ -33,4 +34,20 @@ class RelationChoiceFieldSerializer(DefaultFieldSerializer):
 @adapter(IRelationList, IDexterityContent, Interface)
 @implementer(IFieldSerializer)
 class RelationListFieldSerializer(DefaultFieldSerializer):
-    pass
+    def get_value(self, default=[]):
+        """Return field value reduced to list of non-broken Relationvalues.
+
+        Args:
+            default (list, optional): Default field value. Defaults to empty list.
+
+        Returns:
+            list: List of RelationValues
+        """
+        value = getattr(
+            self.field.interface(self.context), self.field.__name__, default
+        )
+        if not value:
+            return []
+        if isinstance(value, RelationValue):
+            return [value]
```

Comment:
I think this case will not occur since it is a RelationList. I think it can be removed.